app/src/main/res/layout/activity_main.xml

```diff
@@ -9,7 +9,7 @@
     <TextView
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="Hello World!"
+            android:text="My first commit!"
```

Comment:
This looks good, but I think we should put "My second commit!". Remember you already made your first commit, so this is the second one.